docs/how_to/hypervisors/vagrant_libvirt.md

```diff
@@ -0,0 +1,54 @@
+# Vagrant Libvirt #
+
+This driver enables to trigger tests using libvirtd daemon. It is
```

Comment:
should be: This driver enabled _you_ to trigger tests.

Could also be a number of similar things: someone, a tester, etc.